.github/workflows/markdown_crash_test.yaml

```diff
@@ -9,10 +9,12 @@ on:
     branches: [ main ]
     paths:
       - '.github/workflows/markdown_crash_test.yaml'
+      - 'pkgs/markdown/**'
```

Comment:
In the previous PR, I didn't put in the filter on all markdown files, as this test is kind of slow. It used to run only periodically in the old repo, and I switched it to run on changes to the workflow itself. I am fine with always running it, just something to be aware of. I also may be vastly wrong about the running time...

Reply:
Ah, I thought this was inadvertent; I'll revert this change.